doc/radosgw/group.rst

```diff
@@ -0,0 +1,100 @@
+===============
+Groups
+===============
+
+Ceph Object Gateway provides support for Amazon Groups. A group is a collection
+of Users. The Group ARN can be used as a Principal in a Bucket Policy, such that
+the policy applies to all the users belonging to a particular group.
+
+
+Group Management
+====================
+
+A group can be created/ deleted/ updated and users can be added to /removed
```

Comment:
A group may be created, deleted, or updated and users added or removed from a group with `radosgw-admin` commands.